Jenkinsfile

```diff
@@ -6,7 +6,10 @@ pipeline {
         AWS_CREDENTIAL_ID  = 'aws-jenkins'
         AWS_DEFAULT_REGION = 'us-east-1'
         AWS_ECR            = 'public.ecr.aws/oss-presto'
+        AWS_ECR_REG        = 'public.ecr.aws'
+        AWS_ECR_ORG        = 'oss-presto'
```

Comment:
looks like duplicate params.

Reply:
update the script and Jenkinsfile file to use `AWS_ECR` instead. done